server.go

```diff
@@ -31,6 +31,7 @@ import (
 
 	"github.com/monasca/monasca-aggregator/aggregation"
 	"github.com/monasca/monasca-aggregator/models"
+	"github.com/cydev/zero"
```

Comment:
You need to remove this since it is unused. It is also not in glide.yaml

Reply:
done